### PR TITLE
feat: add metadata module in trace-etl

### PR DESCRIPTION
**Pull request by sadadw1**
feat: add metadata module in trace-etl, the main is metadata query, increase, modify and so on #336

### CODE REVIEW

File: trace-etl/trace-etl-metadata/README_CN.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# 概述
```
Comment (goodjava) [inline trace-etl/trace-etl-metadata/README_CN.md:1]: Also add some English documents.

**Comment by goodjava** [comment]
+LGTM

**Review by goodjava — APPROVED** [review]
+LGTM

**Review by wtt40122 — APPROVED** [review]
+LGTM

**Pull request merged**